addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.js

```diff
@@ -97,7 +97,7 @@ const DynamicSnippetProducts = DynamicSnippetCarousel.extend({
             var animation = wSaleUtils.animateClone($navButton, $(ev.currentTarget).parents('.card'), 25, 40);
             Promise.all([fetch, animation]).then(function (values) {
                 wSaleUtils.updateCartNavBar(data);
-                self._render();
+                if (self.actionRerender) self._render();
```

Comment:
`{}` (et `;`) always, be explicit ... 

keep it readable,
avoid error when you will indent it to call a second line
avoid extra diff when you will add a new line...
(avoid bug when you minify)